packages/console/src/components/Steps/index.tsx

```diff
@@ -0,0 +1,36 @@
+import StepItem, { type StepItemProps } from './components/StepItem';
+import { type StepStatus } from './components/StepItem/Status';
+
+export type StepsProps = {
+  stepItems: Array<Pick<StepItemProps, 'title' | 'content' | 'isContentVisibleOnFinished'>>;
+  currentStep: number;
+  currentStatus: StepStatus;
+  classNames?: string;
+};
+
+function Steps({ stepItems, currentStep, currentStatus, classNames }: StepsProps) {
+  const getStepItemStatus = (step: number): StepStatus =>
+    step < currentStep ? 'finished' : step === currentStep ? currentStatus : 'pending';
+
+  return (
+    <div className={classNames}>
+      {stepItems.map((step, index) => {
+        const stepItemStep = index + 1;
```

Comment:
nit: This variable is uncessary and the variable name is quite confusing

Reply:
Rename to `itemStep`, and it will be used to calculate related states about the step.